test/run

```diff
@@ -5,7 +5,7 @@
 make vm
 
 if [ -n "$TEST_SCENARIO" ]; then
-  test/check-cloud TestCloud.test_$TEST_SCENARIO
+  test/check $TEST_SCENARIO
 else
-  test/check-cli
+  test/check TestImages
```

Comment:
Wouldn't it make more sense to specify "TestImages" as a scenario as well? Then this would avoid the entire if condition, and if it's empty, just calling `test/check` should still DTRT and run all tests?

Reply:
See this comment:
https://github.com/cockpit-project/cockpit/pull/11933#discussion_r287764189

We already have the image tests specified as a scenario. I will update the scenarios for better grouping. However if nothing is selected then all tests will be executed which will be quite long. Each of the cloud tests takes around 30-40 mins depending on network speed. IMO it is better to default to something smaller if no scenarios are specified.

Reply:
Agreed about the default if nothing is specified, that's a good point.